lib/features/email/domain/state/download_attachment_for_web_state.dart

```diff
@@ -56,9 +56,12 @@ class DownloadAttachmentForWebSuccess extends UIState {
 class DownloadAttachmentForWebFailure extends FeatureFailure {
 
   final DownloadTaskId? taskId;
+  final Attachment attachment;
```

Comment:
why we need `attachment` here?

Reply:
You're right. Its `blobId` is sufficient. Will change

Reply:
Done